feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -85,14 +93,22 @@ var (
 	}
 
 	atePort3 = attrs.Attributes{
-		Name:    "atePort3",
+		Name:    "port3",
+		MAC:     "02:00:03:01:01:01",
 		IPv4:    "192.0.2.10",
 		IPv4Len: ipv4PrefixLen,
 	}
 )
 
+var inputMap = map[attrs.Attributes]attrs.Attributes{
+	atePort1: dutPort1,
+	atePort2: dutPort2,
+	atePort3: dutPort3,
+}
+
 // configInterfaceDUT configures the interface with the Addrs.
 func configInterfaceDUT(i *telemetry.Interface, a *attrs.Attributes) *telemetry.Interface {
+	deviations.InterfaceEnabled = ygot.Bool(true)
```

Comment:
Let's revert this change because this is an Arista specific deviation.

When running the test against Arista, pass `--deviation_interface_enabled` as a flag.

Reply:
Done